roles/docker/README.md

```diff
@@ -7,6 +7,7 @@ on CentOS 7 or Rocky Linux 8.
 
 | Name                       | Description                                                                                          |
 | -------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `docker_version`           | The version of Docker to install. Defaults to `24.0.9`                                               |
```

Comment:
Do we want to hard code this? Could easily be missed

Reply:
May be worth keeping, though? A similar error could happen again

Reply:
True, but maybe we default to latest? And override to a specific version?

Reply:
yeah defaulting to `latest` makes sense